plugins/polio/serializers.py

```diff
@@ -313,6 +313,7 @@ class Meta:
         ]
 
     modified_by = UserSerializer(required=False, read_only=True)
+    round: Field = serializers.PrimaryKeyRelatedField(read_only=True, many=False)
 
```

Comment:
Yep, this one, that wa smerged yesterday: https://github.com/BLSQ/iaso/pull/704

I think it has been deployed as hotfix already: that's the fix to avoid the crash in reasons for delay